include/libXbSymbolDatabase.h

```diff
@@ -223,6 +261,20 @@ bool XbSymbolScan(const void* xb_header_addr, xb_symbol_register_t register_func
 /// <returns>Return "UNKNOWN" string if does not exist in the database. Otherwise will return library name string.</returns>
 const char* XbSymbolDatabase_LibraryToString(uint32_t library_flag);
 
+/// <summary>
+/// To convert parameter type into string format.
+/// </summary>
+/// <param name="param_type">Input provided param_type from symbol register callback.</param>
+/// <returns>Return "unk" string if does not exist in the database. Otherwise will return parameter type string.</returns>
```

Comment:
```suggestion
/// <returns>Return "unk" string if does not exist in the database. Otherwise it will return the string representation of the parameter type.</returns>
```